### DISCUSSION TITLE
Building software for a small business

### DISCUSSION THREAD

**Discussion by GraceGreen02** [Budibase/budibase#13773] [Help] [answered]
Hi everyone. I am looking to build scheduling and enquiry management software for a horse riding school that they can use in the long run. Budibase seems to be one of the only financially feasible options to do it with. Will I be able to build such an application? I am hesitant because Budibase is new and I do not know if I can trust it, but I want to trust it.

**Comment by JCV67** [comment]
Yes it is feasible. I suggest that along with tables you use Full Challenger plugin, @diogenesbrussels fork is the most complete, allows you to use up to 3 sets and supports several locale.

**Comment by mikesealey** [comment, marked as answer]
Hi @GraceGreen02 

Budibase is a great choice for making tools and apps like booking/scheduling apps, and enquiry management. We appreciate that we are a comparative newcomer but please know that we are here to stay. But don't take my word for it when you can read the reviews of our users and customers [here](https://www.g2.com/products/budibase/reviews). 

Budibase is more than capable of meeting your needs. If you wanted to take a closer look you could play around with some of our templates like [this maintenance schedule app](https://budibase.com/operations/templates/maintenance-schedule-template/) or this [school admin panel](https://budibase.com/operations/templates/school-admin-panel/). Whilst they're not a straight 1-to-1 comparison for your use case, you should be able to recognise the similarities.

Budibase boasts connectivity to a whole host of [external datasources ](https://docs.budibase.com/docs/data-sources) so that you can maintain ownership and _control_ over your data, as well as our own [Budibase DB](https://docs.budibase.com/docs/budibasedb).

Budibase is also an open-source project with a devoted and engaged community surrounding it, with nearly 21k stars on GitHub, and a very active and supportive [Discord server](https://discord.com/invite/ZepTmGbtfF).

I think the best way for you to experience Budibase is _first-hand_. You can get started very quickly, [here](https://account.budibase.app/auth/login), and the [documentation](https://docs.budibase.com/docs/what-is-budibase) will help answer any questions you may have.

If there are any other questions you have, by all means let us know!

**Reply by GraceGreen02** [reply]
Thank you. I have a question about self-hosting. Can I self host by using AWS?
